src/pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -298,7 +298,7 @@ def reset_batch_norm_and_save_state(self, pl_module: "pl.LightningModule") -> No
                 dtype=module.running_var.dtype,
             )
             self.momenta[module] = module.momentum
-            module.momentum = float()
+            module.momentum = None
```

Comment:
This bug was introduced here: https://github.com/Lightning-AI/lightning/pull/13860/files

We should update the type of `self.momenta` accordingly, where values can take Optional[float]